README.md

```diff
@@ -291,3 +291,30 @@ module_test:
   box: centos6
   options: --module-prs qpid/12
 ```
+
+## Client Testing With Docker
+
+The docker/clients directory contains setup and configuration to register clients via subscription-manager using an activation key and start katello-agent. The following are pre-requisites:
+
+ * (docker)[https://docs.docker.com/installation/]
+ * (docker-compose)[https://docs.docker.com/compose/install/]
```

Comment:
I suppose I can, I do think that the docker installation instructions are fairly straightforward and support whatever OS you are using including mentioning which versions it works on (e.g. it has a Fedora section that talks about how you need F20 or greater).